### PR TITLE
Index and show page for features, tests for features.

### PR DESCRIPTION
**Pull request by paigiethefriendlydragon**
Index and show page complete, with tests. Feature model and association model and feature  controller and updated seeds

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
-  arguments = { name: Faker::Address.street_name + " Park", description: Faker::Hipster.paragraph }
-  GreenSpace.create(arguments)
-end
-
-10.times do
-  first_name = Faker::Name.first_name
-  arguments = { first_name: first_name, last_name: Faker::Name.last_name, email: Faker::Internet.safe_email(first_name), password: Faker::Internet.password(8, 12) }
-  User.create(arguments)
-end
+# 20.times do
```
Comment (CassandraNewell) [inline db/seeds.rb]: Please delete all commented-out code

File: spec/models/categorization_spec.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
```
Comment (CassandraNewell) [inline spec/models/categorization_spec.rb]: If there's nothing to test, it's best not to include a test file -- please expand or delete

Comment (CassandraNewell) [reply spec/models/categorization_spec.rb]: note that you could test whether a categorization needs to have a feature/category

File: app/views/features/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<h1 class="text-center"><%= @feature.name %></h1>
```
Comment (CassandraNewell) [inline app/views/features/show.html.erb]: Per the instructions, this page needs to be in React:
> Show page for items (MUST be in React!) that also shows reviews

File: db/seeds.rb (hunk 1 above)
Comment (baker914) [reply db/seeds.rb]: code wasn't meant to be commented out, resubmit will have comments removed

File: spec/models/categorization_spec.rb (hunk 2 above)
Comment (baker914) [reply spec/models/categorization_spec.rb]: tests will be added, and model/db validations

File: app/views/features/show.html.erb (hunk 3 above)
Comment (baker914) [reply app/views/features/show.html.erb]: this is not the item being reviewed the main page for green spaces is in react

**Pull request merged**